db/migrate/20200601101128_add_item_revisions.rb

```diff
@@ -0,0 +1,22 @@
+class AddItemRevisions < ActiveRecord::Migration[5.1]
+  def change
+    create_table "item_revisions", id: false do |t|
+      t.string "uuid", limit: 36, primary_key: true, null: false
+      t.string "item_uuid", limit: 36, null: false
+      t.string "revision_uuid", limit: 36, null: false
+      t.index ["item_uuid"]
+      t.index ["revision_uuid"]
+    end
+
+    create_table "revisions", id: false do |t|
+      t.string "uuid", limit: 36, primary_key: true, null: false
+      t.text "content", limit: 16.megabytes - 1
+      t.string "content_type"
+      t.text "enc_item_key"
```

Comment:
We would need `items_key_id` here as well from the new 004-new brach. This PR should really be targeted at that branch rather than develop.

Reply:
what exactly is the `items_key_id` and how is it different to `item.uuid` ?

Reply:
Recommend reading [this](https://github.com/standardnotes/snjs/blob/004/specification.md) :) items_key_id is the uuid of the items_key item used to encrypt that item.

Reply:
As discussed - we will add this once https://github.com/standardnotes/syncing-server/pull/56 gets merged 

This PR should come after it

Reply:
@karolsojko #56 has been merged :+1:

Reply:
rebased & added `items_key_id` as well to `revision`